### DISCUSSION TITLE
Upgraded to v4, everything breaks

### DISCUSSION THREAD

**Discussion by tenthirtyone** [ProjectOpenSea/seaport-js#543] [General]
I'm jumping from v2 and I am on ethers 5.7. 

Apparently this is no longer valid:
```
 const provider = new providers.Web3Provider(window.ethereum);
 const seaport = new Seaport(provider);
```

And because the protocol has bumped to 1.6 it appears that any orders relying on 1.6 cannot be fulfilled. 

Before assessing LoE to rip everything out and rebuild everything so I can get the same functionality I had two weeks ago and figuring out a way to justify that dev time, can someone confirm this requirement. Because, if so, it's worthwhile for us to spend that time exploring other protocols/options.

**Comment by ryanio** [comment]
you need to upgrade to ethers v6, shouldn't be too difficult: https://docs.ethers.org/v6/migrating/

**Reply by tenthirtyone** [reply]
Alright, then I will explore other options for now.

Thanks
